src/Service.Tests/SqlTests/GraphQLCollationTests/GraphQLCollationTestBase.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.GraphQLCollationTests
+{
+    [TestClass]
+    public class GraphQLCollationTestBase : SqlTestBase
+    {
+        #region Tests
+
+        /// <summary>
+        /// Compares SQL Database Query with GraphQL Query
```

Comment:
Compares the queries to detect what? 
The description should try to answer "What is the goal of the test and how/why does this test achieve that goal?"